ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -100,6 +100,18 @@ else
 	fi
 fi
 
+if ver hwcmp CRAZYFLIE -o ver hwcmp AEROCORE2
```

Comment:
You can use `if ver hwcmp CRAZYFLIE AEROCORE2`.
And the  comment should be updated that the CF2 can optionally have an sd card.

Reply:
@bkueng , Thank you! Updates here: https://github.com/PX4/Firmware/pull/9899/commits

I appreciate your feedback!